pyigm/cgm/utils.py

```diff
@@ -274,4 +274,64 @@ def cgm_from_galaxy_igmsystems(galaxy, igmsystems, rho_max=300*u.kpc, dv_max=400
     # Return
     return cgm_list
 
+def covering_fraction(iontable,colthresh,rhobins=[0,150],returncounts=False,**kwargs):
+    """Given some detection threshold, calculate the ion covering fraction
+    within impact parameter bins
 
+    Parameters
+    ----------
+    iontable : Table
+        Output of CGMAbsSurvey.ion_tbl()
+    colthresh : float or None
+        Detection threshold in log column density; if None, impose no threshold
+    rhobins,optional : list of ints or floats
+        Bins in impact parameter
```

Comment:
If nothing is input, what is done?  Do you just get one 
number for the whole lot?

Reply:
This defaulted to [0,150] kpc, but I like your suggestion (if it was one) better...changed.  I also clarified the units.